conf/config.neon

```diff
@@ -882,6 +883,11 @@ services:
 	-
 		class: PHPStan\Rules\ClassForbiddenNameCheck
 
+	-
+		class: PHPStan\Rules\Classes\ClassConstantRule
+		arguments:
+			checkNonStringableDynamicAccess: %featureToggles.checkNonStringableDynamicAccess%
```

Comment:
Why are you adding the service here in this file? Remove it please.

Reply:
Instead, config.level0.neon should be modified:

1) Remove the rule from `rules` section.
2) Add the service same way you did here.
3) But also add `tags` section with the corresponding tag.